api/src/test/java/uk/gov/dhsc/htbhf/claimant/creator/dwp/entities/uc/UCAdult.java

```diff
@@ -18,25 +21,32 @@
     @JoinColumn(name = "dwp_uc_household_id", nullable = false)
     private UCHousehold household;
 
+    @Size(max = 9)
     @Column(name = "nino")
     private String nino;
 
-    @Column(name = "forename")
-    private String forename;
-
+    @Size(max = 500)
     @Column(name = "surname")
     private String surname;
 
+    @Size(max = 500)
     @Column(name = "address_line_1")
     private String addressLine1;
 
-    @Column(name = "address_line_2")
-    private String addressLine2;
-
-    @Column(name = "address_town_or_city")
-    private String townOrCity;
-
+    @Size(max = 10)
     @Column(name = "address_postcode")
```

Comment:
Any reason why you've added the regexp to email address and phone number but not postcode?

Reply:
Copied directly from DWP...